fdroidserver-0.8-allow-flatpak-aapt.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/fdroidserver/common.py b/fdroidserver/common.py
+index 9800955f..b8c69477 100644
+--- a/fdroidserver/common.py
++++ b/fdroidserver/common.py
+@@ -321,6 +321,7 @@ def find_sdk_tools_cmd(cmd):
+         if os.path.exists(sdk_platform_tools):
+             tooldirs.append(sdk_platform_tools)
+     tooldirs.append('/usr/bin')
++    tooldirs.append('/app/bin')
```

Comment:
This should be fixed upstream. It could be set during build time

Reply:
Let me explain this patch a little bit further. In later versions of _fdroidserver_, _aapt_ isn't required anymore as its functionality is now handled natively by a python library. At Repomaker we so far refused to bump our dependency on _fdroidserver_ from 0.8 to 1.x as it's quite a big change that shortly before the release of Repomaker 1.0.

If you require to have this set upstream, I'll of course follow your request and backport this fix to 0.8. But in my discussion with Persival we decided to not do that as this patch won't be required anymore once we bump _fdroidserver_ to 1.x, which is planned to happen shortly after Repomaker's 1.0.

Reply:
Didn't say that it should happen right now. It would be nice to have those improvements upstream to make things easier to manage here

Reply:
Ah, OK. Thanks for your further explanation.

As I said, it's planned to get rid of _aapt_ soon, so that patch won't be needed here.